docs/multitool-usage.md

```diff
@@ -72,4 +75,5 @@ Run ```Sarif.Multitool convert --help``` for the current list.
 | Name | Purpose |
 | ---- | ------- |
 | pretty-print | Produce pretty-printed JSON output rather than compact form. |
```

Comment:
>form [](start = 72, length = 4)

We should mention this is the default! :) Probably, this argument should be deprecated over time (i.e., we'd only have minify which would reverse our default pretty print behavior).

Reply:
I took a look at our base class and pretty-print isn't default. What we could do is: in a separate PR remove the minify since minify is the opposite of pretty-print. What do you think?

---
In reply to: [529762644](https://github.com/microsoft/sarif-sdk/pull/2160#discussion_r529762644) [](ancestors = 529762644)